dynosaur/tests/pass/multiple-lifetimes.rs

```diff
@@ -0,0 +1,14 @@
+#[dynosaur::dynosaur(DynSomeTrait)]
+trait SomeTrait {
+    async fn multiple_elided_lifetimes(&self, a: &u8, b: &u8);
+    async fn multiple_named_lifetimes<'a, 'b>(&self, a: &'a u8, b: &'b u8, c: fn(&u8));
```

Comment:
Can you include one that repeats the same lifetime parameter on multiple arguments?